docs/DEV_FEE_TECHNICAL_SPEC.md

```
@@ -0,0 +1,438 @@
+
+**Key Design Principles:**
+- Transparent and configurable
+- Non-blocking (failures don't prevent order completion)
+- Full audit trail for accountability
+- Seller-pays model (included in hold invoice)
+
+## Architecture
+
+### Fee Flow Diagram
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)